test/parallel/test-fs-whatwg-url.js

```diff
@@ -72,3 +72,14 @@ if (common.isWindows) {
     message: `File URL host must be "localhost" or empty on ${os.platform()}`
   }));
 }
+
+//path ending with forward slash are not permitted on windows
+if (common.isWindows) {
+  ['%2f', '%2F'].forEach((i) => {
+    fs.readFile(new URL(`file:///c:/tmp/${i}/`), common.expectsError({
+      code: 'ERR_INVALID_FILE_URL_PATH',
+      type: TypeError,
+      message: 'File URL path can not end with / characters'
```

Comment:
Umm... If I'm not mistaken, the `message` option will not match the actual error message.

It should be `'File URL path must not end with / character'`.

By the way, you can use `./configure && make -j4 test` on Unix/MacOS or `vcbuild test` on Windows to run the tests on your computer (For more guides, you can refer to [here](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-6-test)). PR should always pass all the tests.

Reply:
@starkwang  Thanks a lot.